include/mata/inter-aut.hh

```diff
@@ -29,6 +29,8 @@
 #include <string>
 #include <vector>
 
+#include <mata/synchronized_iterator.hh>
```

Comment:
Why is this included here?

Reply:
Hm, if it is not needed, then it can be removed. Or do you mean something else? Hard to tell from your question.

Reply:
You added the include by your PR so I asked what were the reasons. If it was by accident, it can be removed.